lib/mirage/impl/mirage_impl_block.ml

```diff
@@ -260,17 +262,13 @@ let disk =
          only alpanumeric characters)."
       [ "disk" ]
   in
-  Key.(create "disk" Arg.(opt ~stage:`Run string "disk" doc))
+  Key.(create "disk" Arg.(opt string "disk" doc))
 
-let analyze =
-  let doc =
-    Key.Arg.info ~doc:"Analyze at the boot time the given docteur disk."
-      [ "analyze" ]
-  in
-  Key.(create "analyze" Arg.(opt ~stage:`Run bool true doc))
+let analyze = Runtime_key.create "Mirage_runtime.analyze"
 
 let docteur ?(mode = `Fast) ?(disk = disk) ?(analyze = analyze) ?branch
     ?(extra_deps = []) remote =
+  let disk = (disk, Runtime_key.create "Mirage_runtime.disk") in
```

Comment:
TODO: `Mirage_runtime.disk` doesn't exist, we need to decide what we want to do with these runtime `disk` parameters.